lib/checker/checkercomponent.cpp

```diff
@@ -212,11 +212,6 @@ void CheckerComponent::CheckThreadProc()
 
 		lock.unlock();
 
-		if (forced) {
-			ObjectLock olock(checkable);
-			checkable->SetForceNextCheck(false);
-		}
-
```

Comment:
Currently, the attribute is reset before starting the check, so the check plugin can't know the true value of `$force_next_check$`.